src/sql-parser/src/parser.rs

```diff
@@ -4473,6 +4474,23 @@ impl<'a> Parser<'a> {
             options,
         }))
     }
+
+    /// Parse a `RAISE` statement, assuming that the `RAISE` token
+    /// has already been consumed.
+    fn parse_raise(&mut self) -> Result<Statement<Raw>, ParserError> {
+        let severity = match self.parse_one_of_keywords(&[DEBUG, INFO, LOG, NOTICE, WARNING]) {
+            Some(DEBUG) => NoticeSeverity::Debug,
+            Some(INFO) => NoticeSeverity::Info,
+            Some(LOG) => NoticeSeverity::Log,
+            Some(NOTICE) => NoticeSeverity::Notice,
+            Some(WARNING) => NoticeSeverity::Warning,
+            Some(_) | None => {
```

Comment:
nit: `Some(_)` is unreachable, not a client error

Reply:
Hi, I did a bit more of testing, and changing it to unreadable causes statements such as `RAISE BEGIN;` to panic, so I'll keep it as it is.

Reply:
That sounds like there's a bug in the parser then; give me a bit to look into this

Reply:
Ah, sorry, I messed up while testing this, `Some(_)` is indeed unreachable, I just typoed when changing and also mapped `None` to unreachable.